megamek/src/megamek/common/Mech.java

```diff
@@ -2049,7 +2045,8 @@ && getCrew().getOptions().booleanOption(
                     case 11:
                         return new HitData(Mech.LOC_RLEG);
                     case 12:
-                        if (getCrew().hasEdgeRemaining()
+                        if (shouldUseEdge(OptionsConstants.EDGE)
+                                && getCrew().hasEdgeRemaining()
```

Comment:
This looks incorrect; should replace all three lines with

```
if (shouldUseEdge(OptionsConstants.EDGE_WHEN_HEADHIT)) {
```

Reply:
Yep you're right! Fixed in 0821323